backend/src/main/java/com/huellapositiva/domain/model/valueobjects/Email.java

```diff
@@ -41,6 +41,25 @@ public static Email createFrom(ProposalRevisionEmail proposalRevisionEmail, Emai
                 .body(emailTemplate.getParsedTemplate())
                 .build();
     }
+    public static Email createFrom(EmailRecoveryPassword emailRecoveryPassword, EmailTemplate emailTemplate, String from) {
+        validateEmail(emailRecoveryPassword.getEmail());
+        return Email.builder()
+                .from(from)
+                .to(emailRecoveryPassword.getEmail())
+                .subject("Cambio de tu contraseña")
+                .body(emailTemplate.getParsedTemplate())
+                .build();
+    }
+
+    public static Email createFrom(EmailAddress emailAddress, EmailTemplate emailTemplate, String from) {
+        validateEmail(emailAddress.toString());
+        return Email.builder()
+                .from(from)
+                .to(emailAddress.toString())
+                .subject("Confirmacion de cambio de contraseña")
+                .body(emailTemplate.getParsedTemplate())
+                .build();
+    }
```

Comment:
If you pass the subject as an argument or include it in the EmailTemplate object you would get rid of all these extra methods.

Naming "createFrom" to all methods is misleading when you have clearly different implementations. You could name the methods after their intention, e.g. createPasswordConfirmationEmailFrom, createPasswordChangeEmailFrom, ...

Reply:
Mira Ibai, estamos de acuerdo contigo con que el código se repite, pero no solo se repite en esto dos métodos, se repiten en 5 exactamente pero con pequeñas modificaciones. Si te parece bien, consideramos que lo mejor es verlo en el refactor general después de esta pull request porque se tocan cosas más allá del contexto de esta rama.

Reply:
You can decide when you tackle the issue, that is fine for me, just pointing out.